cvat/apps/engine/migrations/0062_delete_previews.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+import traceback
+
+from django.db import migrations
+from django.conf import settings
+from cvat.apps.engine.log import get_migration_logger
+
+def delete_previews(apps, schema_editor):
+    migration_name = os.path.splitext(os.path.basename(__file__))[0]
+    with get_migration_logger(migration_name) as log:
+        def delete_object_previews(db_objects, root_path):
+            for db_obj in db_objects:
```

Comment:
Do we really need to work with DB objects here? Will it be better and faster just remove all root_path/**/preview.jpeg? What is your opinion?